home.php

```diff
@@ -11,9 +11,12 @@
 ?>
 
 <article class="content-area">
-	<?php if ( ! is_front_page() && is_home() && ! empty( $page_for_posts ) ) : ?>
-		<h1 <?php r_page_title_class( '', true ); ?>><?php echo get_the_title( $page_for_posts ); ?></h1>
-	<?php endif; ?>
+	<?php
+	// Output the posts page's title if this is a blog listings page.
+	if ( ! is_front_page() && is_home() && ! empty( $page_for_posts ) ) {
+		responsive_the_title( '<h1 ' . r_page_title_class( '', true ) . '>', '</h1>', true, $page_for_posts );
```

Comment:
Since this title is switched to using the template tag function I think you can remove the parameters from
`r_page_title_class( '', true )` changing it to `r_page_title_class( )` like in the other template tag versions since it doesn't need to be printed now just returned.